cmd/koordlet/options/options.go

```diff
@@ -18,10 +18,46 @@ package options
 
 import (
 	"flag"
+	"net/http"
+	"net/http/pprof"
+	"os"
+	"time"
+
+	"k8s.io/klog/v2"
 )
 
 var (
-	ServerAddr  = flag.String("addr", ":9316", "port of koordlet server")
-	EnablePprof = flag.Bool("enable-pprof", false, "Enable pprof for controller manager.")
-	PprofAddr   = flag.String("pprof-addr", ":9317", "The address the pprof binds to.")
+	ServerAddr   = flag.String("addr", ":9316", "port of koordlet server")
+	EnablePprof  = flag.Bool("enable-pprof", false, "Enable pprof for koordlet.")
+	PprofAddr    = flag.String("pprof-addr", ":9317", "The address the pprof binds to.")
+	KubeAPIQPS   = flag.Float64("kube-api-qps", 40.0, "QPS to use while talking with kube-apiserver.")
```

Comment:
how about keeping the default QPS and Burst the same as the previous config (`20, 30`)

Reply:
QPS and burst have been set to default values.